locations/spiders/blue_bottle_cafe.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+from locations.items import GeojsonPointItem
+
+class BlueBottleCafeSpider(scrapy.Spider):
+
+    name = "bluebottlecafe"
+    allowed_domains = ["www.bluebottlecoffee.com"]
+    start_urls = (
+        'https://bluebottlecoffee.com/api/cafe_search/fetch.json?coordinates=false&query=true&search_value=all',
+    )
+
+    def parse(self, response):
+        results = json.loads(response.body_as_unicode())
+        for region_name in results["cafes"]:
+            for store_data in results["cafes"][region_name]:
+
+                address_string = store_data['address'].replace('\n', ' ').replace('\r', '').replace('<br>', ', ')
+                address_data = address_string.split(", ")
+
+                properties = {
+                    'name': store_data['name'],
+                    'addr:full': address_string,
+                    'addr:city': address_string.split(", ")[1].replace(" ", ""),
```

Comment:
The last `replace()` makes it so two-word cities like "New York" end up squished together like "NewYork".